src/commands/selections.ts

```diff
@@ -326,7 +326,13 @@ export function filter(
   defaultInput?: Argument<string>,
   inverse: Argument<boolean> = false,
   interactive: Argument<boolean> = true,
+  setMainIndex: Argument<boolean> = false,
```

Comment:
I don't really like the idea of adding a parameter which sets the input. Ideally, the input would be enough.

Two alternatives I have in mind:
1. Replace `setMainIndex: true` by `input: "i === " + getCount(_, argument)`. It uses internal implementation details which isn't ideal, but it's reasonable.
   - If it were me, I would modify `buildCommandsExpression` in `load-all.build.ts` to replace `currentCount` by `getCount(_, argument)`, so the expression would now be `input: "i === " + currentCount`, which is less hacky.
2. Add the `count` property to the context passed to the input, such that `input: "i === count"` is valid. For this, you would only have to change line 353 below (`const context = ...`), but it would change the external API.

Reply:
Done